### PR TITLE
chore: Added It Company Web-App

### PR DESCRIPTION
**Pull request by SyedImtiyaz-1**
## Description
It is a Web-App which is called Precious IT Services which I have developed using HTML, CSS, JavaScript and JQuery also it uses **mailJS**-API to Fetch mail on the Run Time.

## Screenshots-

Screenshot from 2023-06-25 00-38-46

### It is super ready to `MERGE` 👍🏻

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "baseUrl": "./src",
     "paths": {
       "@/components/*": ["./components/*"],
+      "@/components/*": ["./components/Contact*"],
```
Comment (priyankarpal) [inline jsconfig.json:7]: why this change?

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -7795,4 +7795,4 @@ packages:
 
 settings:
   autoInstallPeers: true
-  excludeLinksFromLockfile: false
+  excludeLinksFromLockfile: false
```
Comment (priyankarpal) [inline pnpm-lock.yaml:7798]: same, why this change?

File: src/components/Footer.jsx
Comment (priyankarpal) [inline src/components/Footer.jsx:1]: why these changes?

File: src/DB/projects.json [hunk 3]
```diff
@@ -2691,6 +2691,12 @@
         "title": "Portfolio Templet",
         "description": "It's a portfolio templet with functionality.",
         "tech": ["HTML5", "CSS3", "JavaScript"]
+      },
+      {
+        "link": "https://preciousitservices.vercel.app",
```
Comment (priyankarpal) [inline src/DB/projects.json:2696]: add github repository link.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json (hunk 3 above)
Comment (SyedImtiyaz-1) [reply src/DB/projects.json:2696]: sure I'll add gitHub repo

File: jsconfig.json (hunk 1 above)
Comment (SyedImtiyaz-1) [reply jsconfig.json:7]: correcting give me 1min

File: pnpm-lock.yaml (hunk 2 above)
Comment (SyedImtiyaz-1) [reply pnpm-lock.yaml:7798]: pnpm-lock which checks the code using prettier --check pnpm-lock.yaml this chance does not going to reflect any issue in website. Don't worry

**Comment by SyedImtiyaz-1** [comment]
### Resolving Conflict

**Comment by SyedImtiyaz-1** [comment]
@priyankarpal ,
Everything is done you can merge this now. 👍🏻

File: src/DB/projects.json [hunk 4]
```diff
@@ -2678,6 +2678,12 @@
         "title": "Portfolio Templet",
         "description": "It's a portfolio templet with functionality.",
         "tech": ["HTML5", "CSS3", "JavaScript"]
+      },
+      {
+        "link": "https://https://github.com/SyedImtiyaz-1/Precious-IT-Services",
```
Comment (priyankarpal) [inline src/DB/projects.json:2683]: remove  `https://`
```suggestion
        "link": "https://github.com/SyedImtiyaz-1/Precious-IT-Services",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
make sure that your projects have a contributing guide, code of conduct, and licence readme without these requirements I can't merge the PR

**Comment by SyedImtiyaz-1** [comment]
> make sure that your projects have a contributing guide, code of conduct, and licence readme without these requirements I can't merge the PR

I have fixed all the issues you can merge this now .

**Comment by priyankarpal** [comment]
> 

nope without these I can't merge

**Comment by SyedImtiyaz-1** [comment]
> > 
> 
> nope without these I can't merge

So can I raise new issue  ? followed by guidelines
